erizoAPI/ExternalOutput.cc

```diff
@@ -22,6 +23,9 @@ class AsyncCloser : public Nan::AsyncWorker {
     ~AsyncCloser() {}
     void Execute() {
       external_output_->close();
+      while (external_output_->isRecording()) {
```

Comment:
`isRecording` being false does not imply the `ExternalOutput` has finished closing. Consider moving the switch of `recording_` to false at the end of `syncClose`

Reply:
good catch!